cluster/usage/service.go

```diff
@@ -79,6 +95,11 @@ func (m *service) Usage(ctx context.Context) (*types.Report, error) {
 
 				// Only process COLD tenants here
 				if physical.ActivityStatus() == models.TenantActivityStatusCOLD {
+					// Add jitter between cold tenant processing (except for the first one)
+					if len(collectionUsage.Shards) > 0 {
+						m.addJitter()
+					}
```

Comment:
question: Does the fs get overwhelmed at some high number of COLD shards, say > 100, and does fine for ~10-50 shards?  If so, would it make sense to add jitter only every N iterations?

Reply:
this is a percussion not an actual state , so if we have more than 1 we jitter the load